### PR TITLE
Initial schemas pages and documentation, to understand the project

### PR DESCRIPTION
**Pull request by gbalbuena**

### CODE REVIEW

File: schemas.md [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+    process_type o-- imt
+    hazard_type o-- process_type
+    hazard_type o-- event
+    hazard_type o-- event_set
+
+    link hazard_type "/" "To be determined"
+```
+
+
+</details>
```
Comment (liaa2) [inline schemas.md:145]: Do we miss any detail tag here?

Comment (gbalbuena) [reply schemas.md:145]: good catch, shouldn't be there

File: _components/future-content.md [hunk 2]
```diff
@@ -1,8 +1,8 @@
 ---
-title: Future content
-order_number: 3
+name: Future content
+order: 3
 category: homepage
-permalink: /schema.html
+permalink: /components/future-content
```
[the remark this answers is not in the document]
Comment (gbalbuena) [reply _components/future-content.md:5]: actually we can remove that permalink, I make a change in config to autogenerate that part

[the remark this answers is not in the document]
Comment (liaa2) [reply _components/future-content.md]: ok

**Pull request merged**